interpretation/deepseismic_interpretation/azureml_pipelines/pipeline_config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step1":
+    {
+        "type": "MpiStep",
+        "name": "train step",
+        "script": "train.py",
+        "input_datareference_path": "data/",
```

Comment:
No instructions around this parameter - is there a virtual folder in our container with this name?

Reply:
Please add instructions on how to use "azcopy" to copy the data from local FS to BLOB

Reply:
Added a link to AZCopy documentation in commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/986fcb2c2b961aab7c8159846907da635e00fd34